soccer/MainWindow.cpp

```diff
@@ -367,6 +368,9 @@ void MainWindow::updateViews() {
     _processor->logger().getFrames(frameNumber(), _history.size(),
                                    _history.begin());
 
+    _processor->logger().getFrames(frameNumber(), _longHistory.size(),
```

Comment:
How often is this function called? Why are we running getFrames twice (once on the smaller log size, and once on the bigger one) Can these be merged?